core/services/eth/log_broadcaster_test.go

```diff
@@ -160,20 +160,17 @@ func TestLogBroadcaster_ResubscribesOnAddOrRemoveContract(t *testing.T) {
 }
 
 type simpleLogListner struct {
-	handler func(lb ethsvc.LogBroadcast, err error)
-	id      models.ID
+	handler    func(lb ethsvc.LogBroadcast, err error)
+	consumerID *models.ID
 }
 
 func (listner simpleLogListner) HandleLog(lb ethsvc.LogBroadcast, err error) {
 	listner.handler(lb, err)
 }
 func (listner simpleLogListner) OnConnect()    {}
 func (listner simpleLogListner) OnDisconnect() {}
-func (listner simpleLogListner) Consumer() models.LogConsumer {
-	return models.LogConsumer{
-		Type: models.LogConsumerTypeJob,
-		ID:   &listner.id,
-	}
+func (listner simpleLogListner) ConsumerID() *models.ID {
```

Comment:
typo

Reply:
*not a typo... genuine lack of knowledge of the correct spelling of "listener"